src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,88 @@
-export const Pagination = () => {};
+import React from 'react';
+import { getNumbers } from '../../utils';
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (currentPage: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const isSelectedFirstPage = currentPage === 1;
+  const isSelectedLastPage = currentPage === totalPages;
+
+  const handleSelectPrevPage = () => {
+    if (!isSelectedFirstPage) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  const handleSelectNextPage = () => {
+    if (!isSelectedLastPage) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li
+        className={cn('page-item', {
+          disabled: isSelectedFirstPage,
+        })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={isSelectedFirstPage}
+          onClick={handleSelectPrevPage}
```

Comment:
Using an anchor tag with an `href` attribute that contains only a hash can lead to unwanted behavior in the application, such as adding unnecessary entries to the browser's history stack. It's recommended to use a `button` element instead if the link is used for triggering an action, not for navigation.